cobra/flux_analysis/variability.py

```diff
@@ -64,6 +120,55 @@ def calculate_lp_variability(lp, solver, cobra_model, reaction_list,
     return fva_results
 
 
+def _fva_optlang(model, reaction_list, fraction):
+    """Helper function to perform FVA with the optlang interface.
+
+    Parameters
+    ----------
+    model : a cobra model
+    reaction_list : list of reactions
+
+    Returns
+    -------
+    dict
+        A dictionary containing the results.
+    """
+    fva_results = {str(r): {} for r in reaction_list}
+    prob = model.solver.interface
+    with model as m:
+        m.solver.optimize()
+        if m.solver.status != "optimal":
+            raise ValueError("There is no optimal solution "
+                             "for the chosen objective!")
+        # Add objective as a variable to the model than set to zero
+        # This also uses the fraction to create the lower bound for the
+        # old objective
+        v = prob.Variable("fva_old_objective",
```

Comment:
Also tiny details, but personally I much like longer variable names, like here 'fva_old_objective' instead of 'v', when grouped like this, it's easy enough to read but if referred to a couple of lines later then I already forgot what 'v' was